### PR TITLE
html-syntax exercise - Nimrod Castano

### PR DESCRIPTION
**Pull request by nimcastano**
Screenshot 2023-03-27 at 10 17 59 AM

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <head>
+
+    <title>HTML Syntax</title>
+
+  </head>
+  <body>
+
+
+    <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png
+">
```
Comment (shawnkost) [inline html-syntax/index.html:12]: Every `img` should have an `alt` attribute.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Nimrod!

File: html-syntax/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 
     <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png
-">
+" alt="Pikachu is as overrated as it gets.">
```
Comment (shawnkost) [inline html-syntax/index.html:12]: I definitely agree, but the `alt` tag should be descriptive of what the actual image is. The main purpose of this attribute is to provide people with vision impairments or other disabilities, extra information on what's appearing on the page. Something as simple as "Pikachu" would be sufficient.

Comment (nimcastano) [reply html-syntax/index.html]: ahhhh okay lol thank you

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**